src/components/JobForm/JobForm.js

```diff
@@ -38,10 +42,11 @@ const JobForm = ({ updateJobAddedStatus }) => {
   const handleSubmit = (e) => {
     e.preventDefault()
     const newJob = input
-    postNewJob(newJob)
+    postNewJob(newJob, user.id)
     .then(() => {
       updateJobAddedStatus(true)
       updateStatus(true)
+      history.push('/homepage')
```

Comment:
If we're using history.push() instead of the Redirect we can remove the `updateStatus()` hook. Its only purpose is to trigger a redirect.